node/node_v2.go

```diff
@@ -34,7 +34,8 @@ type RawBundles struct {
 }
 
 func (n *Node) DownloadBundles(ctx context.Context, batch *corev2.Batch, operatorState *core.OperatorState) ([]*corev2.BlobShard, []*RawBundles, error) {
-	if n.RelayClient == nil {
+	relayClient, ok := n.RelayClient.Load().(clients.RelayClient)
```

Comment:
will this panic if Load() returns nil?

Reply:
No. In that case, `relayClient` will be `nil` and `ok` will be false